src/aurora.ts

```diff
@@ -388,11 +394,22 @@ export class Aurora extends Construct {
     this.secrets = secrets.map((s) => s.secret);
 
     if (!props.skipProxy) {
+      if (props.proxySecurityGroups) {
+        this.proxySecurityGroups = props.proxySecurityGroups;
+      } else {
+        this.securityGroups = [
```

Comment:
i think you mean `this.proxySecurityGroups` ?

Reply:
Ugh. yes. Thank you. Tests added.

Reply:
super excited i caught a bug before coffee 🥳